pennylane/ops/qubit.py

```diff
@@ -1048,6 +1048,14 @@ class CRX(Operation):
     num_wires = 2
     par_domain = "R"
     grad_method = "A"
+
+    # gradient recipe
+    c1 = (np.sqrt(2) - 4 * np.cos(np.pi / 8)) / (4 - np.cos(np.pi / 8))
+    c2 = (np.sqrt(2) - 1) / (4 * np.cos(np.pi / 8) - 2)
```

Comment:
Can these be written in terms of `a` and `b`?

Reply:
No unfortunately, the equation is implicit :(

![image](https://user-images.githubusercontent.com/2959003/99816996-51ed0800-2b87-11eb-8da7-bc20ed889035.png)

This is just one solution given the below values of a and b